src/plugins/d3/renderer/hooks/useSeries/prepareSeries.ts

```diff
@@ -76,15 +45,10 @@ export function prepareSeries(args: {
             return prepareBarYSeries({series: series as BarYSeries[], legend, colorScale});
         }
         case 'scatter': {
-            return series.reduce<PreparedSeries[]>((acc, singleSeries, index) => {
-                acc.push(
-                    ...prepareAxisRelatedSeries({series: singleSeries, legend, colorScale, index}),
-                );
-                return acc;
-            }, []);
+            return prepareScatterSeries({series: series as ScatterSeries[], legend, colorScale});
         }
         case 'line': {
-            return prepareLineSeries({
+            return prepareLine({
```

Comment:
Why in 'scatter' case used `prepareScatterSeries` and `prepareLine` used in 'line'? It is about naming